pyvera/__init__.py

```diff
@@ -241,9 +242,30 @@ def get_changed_devices(self, timestamp):
         payload.update({
             'id': 'lu_sdata',
         })
-        result = self.data_request(payload, TIMEOUT*2).json()
+        
+        LOG.debug("get_changed_devices() requesting payload %s", str(payload))
+        r = self.data_request(payload, TIMEOUT*2)
+        r.raise_for_status()
+
+        # Unfortunately, not all JSON implementations return the well-known
+        # json.decoder.JSONDecodeError, so attempt to trap and raise what
+        # the poller expects.
+        try:
+            result = r.json()
+        except json.decoder.JSONDecodeError:
+            raise # pass it up
+        except Exception as ex:
+            raise requests.RequestException("JSON decode error: " + str(ex))
```

Comment:
This is really a exception of our own devising - rather than a `RequestException` - so I think it would be better to define an exception of our own - rather than repurposing an existing exception.

They are easy to define
```
class VeraDecodeException(Exception):
    pass
```
Although you'll need to import it into `subscript.py` so you can catch it.

Happy to help if needed.